pkg/operator/controller/status.go

```diff
@@ -243,10 +246,6 @@ func (r *reconciler) computeOperatorProgressingCondition(oldCondition *configv1.
 				messages = append(messages, fmt.Sprintf("Moving to openshift-cli image version %q.", r.OpenshiftCLIImage))
 				progressing = true
 			}
-		default:
-			if oldVersion, ok := oldVersionsMap[opv.Name]; ok && oldVersion != opv.Version {
-				messages = append(messages, fmt.Sprintf("Upgraded %s to %q.", opv.Name, opv.Version))
-			}
```

Comment:
https://github.com/openshift/cluster-dns-operator/blob/7fc741b2e4abe724457ecee73d8297dc4c02fd09/pkg/operator/controller/status.go#L252 may not be sufficient to print 'Upgraded ...' msg in all cases. I think we need:
```
if progressing {
	progressingCondition.Status = configv1.ConditionTrue
	progressingCondition.Reason = "Reconciling"
} else {
	progressingCondition.Status = configv1.ConditionFalse
	progressingCondition.Reason = "AsExpected"
}

if len(messages) > 0 {
	progressingCondition.Message = strings.Join(messages, "\n")
} else {
	progressingCondition.Message = "Desired and available number of DNS DaemonSets are equal"
}
```

Reply:
Thanks! Fixed.